src/features/profile/components/atoms/ProfilePicture.tsx

```diff
@@ -0,0 +1,56 @@
+import Image from 'next/image';
+
+import { useAtomValue } from 'jotai';
+import { Pencil } from 'lucide-react';
+
+import { Button } from '@/components/ui/button';
+
+import { profileModeAtom } from '../../stores/profile-atoms';
+
+interface ProfilePictureProps {
+  src: string;
+}
+
+const ImageContainer = ({ children }: { children: React.ReactNode }) => (
+  <div className="relative h-[128px] w-[128px] overflow-hidden rounded-[64px] border border-[#e2e8f0]">
+    {children}
+  </div>
+);
+
+const StyledImage = ({ src }: { src: string }) => (
+  <Image
+    src={src}
+    alt="profile picture"
```

Comment:
```suggestion
    alt="{`${userName}'s profile picture`}"
```

이거 접근성 너무 구짐. 수정해주세요.